scaleway/helpers_instance.go

```diff
@@ -78,6 +78,20 @@ func orderVolumes(v map[string]*instance.Volume) []*instance.Volume {
 	return orderedVolumes
 }
 
+// orderServerVolumes return an ordered slice based on the volume map key "0", "1", "2",...
+func orderServerVolumes(v map[string]*instance.VolumeServer) []*instance.VolumeServer {
```

Comment:
Why is a new function needed?

Reply:
This method is similar but manage a different type